WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -200,8 +200,14 @@ def deletePilots( self, pilotIDs, conn = False ):
 
     if type( pilotIDs ) != type( [] ):
       return S_ERROR( 'Input argument is not a List' )
+    
+    for pilotID in pilotIDs:
```

Comment:
flag here

Reply:
Yes and no. It's pretty awful place, one DB class would need to access another DB. Luckily, all cases seem to be covered on service level (like in WMSAdministratorHandler above, calls to delete pilot, then delete logging). I'm going to remove this part of code.

Reply:
yes, good point.